scripts/bld

```diff
@@ -2,61 +2,15 @@
 
 # A script to build and install the C++ library
 
-# Supported types as of 2022-09-21:
-# * Release
-# * Debug
-# * RelWithDebInfo
-# * Coverage
+set -eu -o pipefail
 
-BUILD_TYPE=${1:-Release} # default to Release build
-if [ ! -z "$LIBTILEDBSOMA_DEBUG_BUILD" ]; then
-  echo
-  echo ================================================================
-  echo LIBTILEDBSOMA SCRIPTS/BLD BUILD_TYPE ${BUILD_TYPE}
-  echo ================================================================
-else
-  echo "Building ${BUILD_TYPE} build"
-fi
```

Comment:
I found the output from this verbose setup _far_ easier to parse -- while it lasted. :( 

👋 ✌️ 🖖

Reply:
I found all of the debug messages made the code harder to modify/maintain.

Reply:
@gspowley which I knew would happen hence 🖖